src/plugins/index_pattern_field_editor/public/components/flyout_panels/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Panels } from './flyout_panels';
+import { Panel } from './flyout_panel';
+import { Header } from './flyout_panels_header';
+import { Footer } from './flyout_panels_footer';
+
+export const Flyout = {
```

Comment:
Can you name this something more specific? I worry about other engineers mistakenly importing this component vs the standard EuiFlyout because of auto-complete.